### PR TITLE
Beer and Tell blog post for August 2016.

### PR DESCRIPTION
**Pull request by Osmose**
@pmclanahan @groovecoder @TatumCreative @rehandalal r?

I'll be posting this on Friday morning, so please leave your feedback before then. Thanks!

### CODE REVIEW

File: beer_and_tell/2016-08-August.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+[Osmose]: https://mozillians.org/en-US/u/Osmose/
+[PyJEXL]: https://github.com/Osmose/pyjexl
+[JEXL]: https://github.com/TechnologyAdvice/Jexl
+
+## pmac: rtoot.org
+Next up was [pmac][], who shared [rtoot.org][], the website for "The Really
+Terrible Orchestra of the Triangle". It's a static site for pmac's local
+orchestra built using [lektor][], an easy-to-use static site generator. One of
+the more interesting features pmac highlighted was lektor's [data model][]. He
+showed a lektor plugin he wrote that added support for CSV fields as part of the
```
Comment (pmac) [inline beer_and_tell/2016-08-August.md]: Please link `lektor plugin` to https://github.com/pmac/rtoot/blob/master/packages/csv-field/lektor_csv_field.py

**Comment by pmac** [comment]
Just that one note. Thanks Osmose! r+

**Comment by groovecoder** [comment]
r+ thanks!

**Comment by rehandalal** [comment]
lgtm.

**Pull request merged**